src/main/java/uk/gov/ons/ssdc/supporttool/config/WebSecurityConfig.java

```diff
@@ -26,30 +30,34 @@ public class WebSecurityConfig extends WebSecurityConfigurerAdapter {
   @Value("${CSP-upgrade-policy}")
   private String cspUpgradePolicy;
 
-  @Override
-  protected void configure(HttpSecurity http) throws Exception {
-    http.headers()
-        .defaultsDisabled()
-        .frameOptions()
-        .deny()
-        .contentTypeOptions()
-        .and()
-        .httpStrictTransportSecurity()
-        .requestMatcher(AnyRequestMatcher.INSTANCE)
-        .includeSubDomains(true)
-        .maxAgeInSeconds(31536000)
-        .and()
-        .referrerPolicy(ReferrerPolicyHeaderWriter.ReferrerPolicy.NO_REFERRER)
-        .and()
-        .contentSecurityPolicy(
-            String.format(
-                "default-src 'self'; manifest-src %s ; style-src 'self' 'unsafe-inline' ; %s block-all-mixed-content",
-                TRUSTED_CDN_DOMAIN, cspUpgradePolicy))
-        .and()
-        .permissionsPolicy()
-        .policy(
-            "accelerometer=(),autoplay=(),camera=(),display-capture=(),document-domain=(),encrypted-media=(),fullscreen=(),geolocation=(),gyroscope=(),magnetometer=(),microphone=(),midi=(),payment=(),picture-in-picture=(),publickey-credentials-get=(),screen-wake-lock=(),sync-xhr=(self),usb=(),xr-spatial-tracking=()");
-
-    http.csrf().disable();
+  @Bean
+  public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
+    http.headers(
+        headers ->
+            headers
+                //                .defaultsDisabled()
+                .contentSecurityPolicy(
+                    cps ->
+                        cps.policyDirectives(
+                            String.format(
+                                "default-src 'self'; manifest-src %s ; style-src 'self' 'unsafe-inline' ; %s block-all-mixed-content",
+                                TRUSTED_CDN_DOMAIN, cspUpgradePolicy)))
+                .frameOptions(frameOptionsConfig -> frameOptionsConfig.deny())
+                .httpStrictTransportSecurity(
+                    hstsConfig ->
+                        hstsConfig
+                            .includeSubDomains(true)
+                            .maxAgeInSeconds(31536000)
+                            .requestMatcher(AnyRequestMatcher.INSTANCE))
+                .referrerPolicy(
+                    referrerPolicyConfig ->
+                        referrerPolicyConfig.policy(
+                            ReferrerPolicyHeaderWriter.ReferrerPolicy.NO_REFERRER))
+                .permissionsPolicy(
+                    permissionsPolicyConfig ->
+                        permissionsPolicyConfig.policy(
+                            "accelerometer=(),autoplay=(),camera=(),display-capture=(),document-domain=(),encrypted-media=(),fullscreen=(),geolocation=(),gyroscope=(),magnetometer=(),microphone=(),midi=(),payment=(),picture-in-picture=(),publickey-credentials-get=(),screen-wake-lock=(),sync-xhr=(self),usb=(),xr-spatial-tracking=()")));
+    http.csrf(AbstractHttpConfigurer::disable);
```

Comment:
```suggestion
        headers -> headers.contentSecurityPolicy(
                    cps -> cps.policyDirectives(
                        String.format(
                            "default-src 'self'; manifest-src %s ; style-src 'self' 'unsafe-inline' ; %s block-all-mixed-content",
                            TRUSTED_CDN_DOMAIN, cspUpgradePolicy)
                    )
            )
            .frameOptions(frameOptionsConfig -> frameOptionsConfig.deny())
            .httpStrictTransportSecurity(
                hstsConfig -> hstsConfig
                    .includeSubDomains(true)
                    .maxAgeInSeconds(31536000)
                    .requestMatcher(AnyRequestMatcher.INSTANCE))
            .referrerPolicy(
                referrerPolicyConfig -> referrerPolicyConfig.policy(
                    ReferrerPolicyHeaderWriter.ReferrerPolicy.NO_REFERRER
                )
            )
            .permissionsPolicy(permissionsPolicyConfig -> permissionsPolicyConfig.policy(
                "accelerometer=(),autoplay=(),camera=(),display-capture=(),document-domain=(),encrypted-media=(),fullscreen=(),geolocation=(),gyroscope=(),magnetometer=(),microphone=(),midi=(),payment=(),picture-in-picture=(),publickey-credentials-get=(),screen-wake-lock=(),sync-xhr=(self),usb=(),xr-spatial-tracking=()"
                )
            )
    );
    
    http.csrf(AbstractHttpConfigurer::disable);
```
It just seemed really unreadable to me? You don't have to change it if you prefer yours/think it's not really a problem

Reply:
The way it currently is, is formatted by maven so if I changed it to that, it would get changed back